tee-worker/core-primitives/stf-executor/src/error.rs

```diff
@@ -86,3 +88,9 @@ impl From<itp_node_api::metadata::Error> for Error {
 		Self::NodeMetadata(e)
 	}
 }
+
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
+pub(crate) struct ErrorResponse<T: Encode> {
```

Comment:
do you think it's better to go to `trusted_call_rpc_response.rs`?

Reply:
And shall we always position `req_ext_hash` as the last field to be consistent with other `*Response` structs?

Reply:
If we make `Response` generic then it will be enforced
```
#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
pub(crate) struct Response<T: Encode> {
	pub req_ext_hash: H256,
	pub value: T,
}
```